File: 012)CatchTheOnePiece/app/src/main/java/com/example/catchtheonepiece/ApiUtils.java
package com.example.catchtheonepiece;

public class ApiUtils {
    public static  final  String BASE_URL = "https://elyesasimsek.com/";
    public static  CharacterDAOInterface getFilmlerDaoInterface(){
        return RetrofitClient.getClient(BASE_URL).create(CharacterDAOInterface.class);
    }
}

File: 053)CatchTheOnePieceKotlin/app/src/main/java/com/elyesasimsek/catchtheonepiecekotlin/ApiUtils.kt
package com.elyesasimsek.catchtheonepiecekotlin

class ApiUtils {
    companion object{
        const val BASE_URL = "https://elyesasimsek.com/"

        fun getCharacterDAOInterface(): CharacterDAOInterface {
            return RetrofitClient.getClient(BASE_URL).create(CharacterDAOInterface::class.java)
        }
    }
}